workflows/github/versions/0.0.3/README.md

```diff
@@ -0,0 +1,18 @@
+# Github
+
+## Summary
+
+A set of templates to perform operations against GitHub such as reporting a commit status check, creating a pull request, and extracting webhook variables.
+
+## Templates
+
+1. [commit-status](https://github.com/codefresh-io/argo-hub/blob/main/workflows/github/versions/0.0.3/docs/commit-status.md)
+1. [create-pr](https://github.com/codefresh-io/argo-hub/blob/main/workflows/github/versions/0.0.3/docs/create-pr.md)
+1. [create-pr-comment](https://github.com/codefresh-io/argo-hub/blob/main/workflows/github/versions/0.0.3/docs/create-pr-comment.md)
+1. [extract-webhook-variables](https://github.com/codefresh-io/argo-hub/blob/main/workflows/github/versions/0.0.3/docs/extract-webhook-variables.md)
```

Comment:
This will render correctly as a numbered list. See the rendered view: https://github.com/codefresh-io/argo-hub/blob/9892041b310bd12b0f9be55f45cdc780914a176c/workflows/github/versions/0.0.3/README.md